json-endpoints/jetpack/class.jetpack-json-api-get-post-backup-endpoint.php

```diff
@@ -3,12 +3,12 @@
 class Jetpack_JSON_API_Get_Post_Backup_Endpoint extends Jetpack_JSON_API_Endpoint {
 	// /sites/%s/posts/%d/backup      -> $blog_id, $post_id
 
-	protected $needed_capabilities = array();
+	protected $needed_capabilities = array(); // This endpoint is only accessible using a site token
 	protected $post_id;
 
 	function validate_input( $post_id ) {
 		if ( empty( $post_id ) || ! is_numeric( $post_id ) ) {
-			return new WP_Error( 'post_id_not_specified', __( 'You must specify a Post ID', 'jetpack' ) );
+			return new WP_Error( 400, __( 'You must specify a Post ID', 'jetpack' ) );
```

Comment:
Sorry I wasn't specific 
I think it needs to change to 
WP_Error( 'post_id_not_specified', __( 'You must specify a Post ID', 'jetpack' ), 400 );